collect_app/src/main/java/org/odk/collect/android/activities/DataManagerList.java

```diff
@@ -72,6 +72,7 @@ public void onClick(View v) {
                         Integer.toString(mSelected.size()));
                 if (mSelected.size() > 0) {
                     createDeleteInstancesDialog();
+                    ListViewUtils.disableToggleButton(mToggleButton, getListView());
```

Comment:
This does not work as intended because the dialog creation is non-blocking and the code written after `createDeleteInstancesDialog()` is run immediately on dialog creation rather than after the dialog disappears.

Additionally, this method name is misleading. It suggests that the method will always disable the toggle button but in fact, it checks to see if there are still items in the `ListView` and disables it if not.

I think that in this case it is clearer to write 
````
if (getListView().getCount() == 0) {
    mToggleButton.setEnabled(false);
}
````

You will need to move this into the dialog yes/no listener for it to happen at the correct time (after instances have been deleted).

I also think it would be good to have that code when the view is first created. Right now if there are no forms at all when I get to the delete saved forms view, I end up with an active "Select All" button which I can click to change to "Toggle All"!